packages/e2e-tests/register/models/SetupSeedPhrase.js

```diff
@@ -0,0 +1,16 @@
+class SetupSeedPhrasePage {
+    constructor(page) {
+        this.page = page;
+    }
+    async navigate(accountId) {
+        await this.page.goto(`/setup-seed-phrase/${accountId}/phrase`);
+    }
+    async copySeedPhrase() {
+        await this.page.click(`button:text-matches("Copy Phrase", "i")`);
```

Comment:
test id please - we don't want this test to fail if we simply rename `Copy Phrase` to `Copy Seed Phrase` :)